src/google/pyproject.toml

```diff
@@ -0,0 +1,48 @@
+[tool.poetry]
+name = "google-extractor"
+version = "0.1.1"
+description = ""
+authors = ["Brad Banister <brad@doublelinepartners.com>"]
+packages = [
+    { include = "google-extractor" }
+]
+
+[tool.poetry.dependencies]
+python = "^3.8"
+google-api-python-client = "^1.11.0"
+google-auth-oauthlib = "^0.4.1"
+jupyter = "^1.0.0"
+matplotlib = "^3.3.1"
+numpy = "^1.19.1"
+pandas = "^1.1.1"
+python-dotenv = "^0.14.0"
+seaborn = "^0.10.1"
+toolz = "^0.10.0"
+
+[tool.poetry.dev-dependencies]
+pytest = "^5.2"
+pylint = "^2.6.0"
+black = "^20.8b1"
+rope = "^0.17.0"
+flake8 = "^3.8.3"
+
+[[tool.poetry.source]]
+name = "myget"
+url = "https://www.myget.org/F/ed-fi/python/"
+secondary = true
+
+[build-system]
+requires = ["poetry>=1.0.10"]
+build-backend = "poetry.masonry.api"
+
+## [npm-like scripts as comments]
+##
+## configure poetry for publication to (temporary) myget feed:
+# poetry config repositories.myget https://www.myget.org/F/ed-fi/python/
```

Comment:
No problem leaving this right now. We'll never publish python to MyGet. Will publish to Azure Artifacts instead, see https://techdocs.ed-fi.org/display/EDFIODS/Azure+Artifacts+Setup

Reply:
Yes, I was testing it out both to evaluate publishing as a potential distribution mechanism and to see if Poetry made it as easy as claimed, especially for a private feed scenario. It does. 👍